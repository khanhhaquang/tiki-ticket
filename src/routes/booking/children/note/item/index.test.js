import React from 'react';
import renderer from 'react-test-renderer';
import { mount } from 'enzyme';
import Item from './index';

describe('Test Booking/Note/Item component', () => {
	test('Snapshot note item without type and content', () => {
		const component = renderer.create(<Item />);
		const tree = component.toJSON();
		expect(tree).toMatchSnapshot();
	});

	test('Snapshot note item with type not right', () => {
		const component = renderer.create(<Item type='asdfsdf' />);
		const tree = component.toJSON();
		expect(tree).toMatchSnapshot();
	});

	const component = mount(<Item type='booked' content='fsadfasd' />);
	test('Check item content', () => {
		expect(component.find('.content')).toBeTruthy();
	});
});
