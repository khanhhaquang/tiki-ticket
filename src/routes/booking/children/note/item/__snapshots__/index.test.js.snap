// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Booking/Note/Item component Snapshot note item with type not right 1`] = `
<div
  className="sc-bdVaJa jHVvyK item"
>
  <span
    className="pre asdfsdf"
  />
  <span
    className="content"
  />
</div>
`;

exports[`Test Booking/Note/Item component Snapshot note item without type and content 1`] = `
<div
  className="sc-bdVaJa jHVvyK item"
>
  <span
    className="pre undefined"
  />
  <span
    className="content"
  />
</div>
`;
